public/modules/custom/helfi_rekry_content/src/Entity/HelJobListing.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Drupal\helfi_rekry_content\Entity;
+
+use Drupal\node\Entity\Node;
+
+/**
+ * Bundle class for hel_map paragraph.
+ */
+class HelJobListing extends Node {
+  /**
+   * Constructor.
+   *
+   * @param array $values
+   *   Values that the entity contains.
+   * @param $entity_type
+   *   Entity type.
+   * @param $bundle
+   *   Entity bundle.
+   * @param $translations
+   *   Entity translations.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  public function __construct(array $values, $entity_type, $bundle = FALSE, $translations = []) {
+    parent::__construct($values, $entity_type, $bundle, $translations);
+
+    $this->storage = \Drupal::entityTypeManager()
```

Comment:
Use `$this-entityTypeManager()` instead and we shouldn't populate this in constructor, otherwise this gets initialized on every page load for no reason.

Reply:
Changed this.